test/e2e/framework/rc/.import-restrictions

```diff
@@ -1,9 +1,13 @@
 # This E2E framework sub-package is currently allowed to use arbitrary
-# dependencies, therefore we need to override the restrictions from
-# the parent .import-restrictions file.
+# dependencies except of k/k/pkg, therefore we need to override the
+# restrictions from the parent .import-restrictions file.
 #
 # At some point it may become useful to also check this package's
 # dependencies more careful.
 rules:
+  - selectorRegexp: "^k8s[.]io/kubernetes/pkg"
+    allowedPrefixes:
+    - k8s.io/kubernetes/pkg/api/v1/pod
```

Comment:
Same here.

Reply:
... and below.

Reply:
I'd suggest to leave k8s.io/kubernetes/pkg/api/v1/pod as this the only dependency of this submodule. If we remove it we'll basically allow to add more deps. The overall idea of this PR was to add such a control.

Reply:
Okay.